### ISSUE TITLE
Python Script not working.

### ISSUE THREAD

**Issue by Hennard** [MISP/misp-dashboard#160]
./diagnostic.py
Dependency not met. Either not in a virtualenv or dependency not installed.
-Error: No module named 'redis'

Can run the redis module in python on the PC, but not with the script.

**Comment by whoisroot** [comment]
Did you activate the venv (`source DASHENV/bin/activate`) before running?

**Comment by Hennard** [comment]
Hi, I run **start_all.sh** and all started up with no errors.

**Comment by whoisroot** [comment]
Yes, it sources the venv and start subprocesses using the venv, but to run the `diagnostic.py` you have to either source the venv for the current session or have the dependencies installed globaly.

**Comment by Hennard** [comment]
Hi,
I redid the install and use this setup: https://kmhcorp.com/installing-misp-dashboard/
Dashboard turns green for Misp1 and then turn red when the feed's start updating.

Is the following right:

[RedisGlobal]
host=192.168.22.128
port=6379
misp_web_url = https://192.168.177.133
misp_instances = [{
    "name": "misp",
    "url": "https://192.168.177.133",
    "zmq": "tcp://192.168.177.133:50000"}]

**Comment by Hennard** [comment]
Hi, I restarted zmq, now getting this error:

host:/var/www/misp-dashboard$ ./diagnostic.py
✔ Virtual environment and packages
✔ Configuration
✔ File permission
✔ Redis
✔ Zmq
✔ Processes status: Both processes are running
✖ Subscriber status: zmq_subscriber seems not to be working.
        ➥ Consider restarting it: 1169 /var/www/misp-dashboard/DASHENV/bin/python ./zmq_subscriber.py -n misp1 -u tcp://localhost:50000
